packages/mongodb-rag-ingest/src/embed/updateEmbeddedContent.ts

```diff
@@ -20,13 +22,15 @@ export const updateEmbeddedContent = async ({
   sourceNames,
   embedder,
   chunkOptions,
+  embedConcurrencyOptions,
 }: {
   since: Date;
   embeddedContentStore: EmbeddedContentStore;
   pageStore: PageStore;
   embedder: Embedder;
   chunkOptions?: Partial<ChunkOptions>;
   sourceNames?: string[];
+  embedConcurrencyOptions?: ConcurrencyOptions["embed"];
```

Comment:
style: please call this property just `concurrencyOptions`. the fact that it's on the function `updateEmbeddedContent()`, makes the *embed* part implicitly redundant


```suggestion
  concurrencyOptions?: ConcurrencyOptions["embed"];
```

Reply:
also, i think better interface design would be to define `EmbedConcurrencyOptions` in this file, ie:

```ts
export interface EmbedConcurrencyOptions {
 
    /**
      Number of pages to chunk and embed concurrently.
     */
    processPages?: number,
    /**
      Maximum number of chunks per page to generate chunks for concurrently.
      This includes the creation of the chunk embeddings and any chunk preprocessing.
     */
    createChunks?: number,
}
```

and then use that here, such as:

```suggestion
  concurrencyOptions?: EmbedConcurrencyOptions;
```